gradle.properties

```diff
@@ -15,3 +15,10 @@ org.gradle.jvmargs=-Xmx1536m
 # This option should only be used with decoupled projects. More details, visit
 # http://www.gradle.org/docs/current/userguide/multi_project_builds.html#sec:decoupled_projects
 # org.gradle.parallel=true
+artifactory_user=""
```

Comment:
can you please explain this a bit? does this change something? do we need to use the credentials in publishing.gradle then?

Reply:
yes, these are credentials for the artifactory, provided by jenkins